Fixes:

/var/tmp/fst/src/ncurses-1.0.0/ncurses_fe.c:31: error: duplicate 'static'
/var/tmp/fst/src/ncurses-1.0.0/ncurses_fe.c:36: error: duplicate 'static'
/var/tmp/fst/src/ncurses-1.0.0/ncurses_fe.c:43: error: duplicate 'static'
/var/tmp/fst/src/ncurses-1.0.0/ncurses_fe.c:49: error: duplicate 'static'
/var/tmp/fst/src/ncurses-1.0.0/ncurses_fe.c:56: error: duplicate 'static'
make: *** [ncurses_fe.lo] Error 1

diff -Naur ncurses-1.0.0.orig/ncurses_fe.c ncurses-1.0.0/ncurses_fe.c
--- ncurses-1.0.0.orig/ncurses_fe.c	2009-08-08 00:18:48.000000000 +0200
+++ ncurses-1.0.0/ncurses_fe.c	2009-08-08 00:19:03.000000000 +0200
@@ -27,32 +27,32 @@
 
 #if ZEND_MODULE_API_NO >= 20071006
 // No more define with PHP 5.3.0
-static 
+ 
 	ZEND_BEGIN_ARG_INFO(first_arg_force_ref, 0)
 		ZEND_ARG_PASS_INFO(1)
 	ZEND_END_ARG_INFO();
 
-static 
+ 
 	ZEND_BEGIN_ARG_INFO(second_arg_force_ref, 0)
 		ZEND_ARG_PASS_INFO(0)
 		ZEND_ARG_PASS_INFO(1)
 	ZEND_END_ARG_INFO();
 #endif
 
-static 
+ 
 	ZEND_BEGIN_ARG_INFO(firstandsecond_args_force_ref, 0)
 		ZEND_ARG_PASS_INFO(1)
 		ZEND_ARG_PASS_INFO(1)
 	ZEND_END_ARG_INFO();
 
-static
+
 	ZEND_BEGIN_ARG_INFO(secondandthird_args_force_ref, 0)
 		ZEND_ARG_PASS_INFO(0)
 		ZEND_ARG_PASS_INFO(1)
 		ZEND_ARG_PASS_INFO(1)
 	ZEND_END_ARG_INFO();
 
-static 
+ 
 	ZEND_BEGIN_ARG_INFO(second_thru_fourth_args_force_ref, 0)
 		ZEND_ARG_PASS_INFO(0)
 		ZEND_ARG_PASS_INFO(1)
